src/editor/edit.js

```diff
@@ -127,6 +130,14 @@ const VideoEditSettings = (props) => {
                         buttonChangeLabel="Change thumbnail image"
                         {...props}
                     />
+                    <SelectControl
+                      label={__('Caption Language Preference', 'ghwp')}
+                      value={ccLangPref}
+                      options={ccLangPrefOptions}
+                      onChange={(ccLangPref) => {
+                          setAttributes({ ccLangPref }); // @TODO: Add functionality to save this value
```

Comment:
I don't understand this TODO

Reply:
@AndreasMaros : That referred to the question in Slack. I thought the setAttributes function was not yet available here.